jbpm-designer-backend/src/main/java/org/jbpm/designer/web/preprocessing/impl/JbpmPreprocessingUnit.java

```diff
@@ -618,9 +644,15 @@ public void evaluateWorkDefinitions(Map<String, WorkDefinitionImpl> workDefiniti
 
                 Map<String, Object> parameterValues = new HashMap<>();
                 if (workDefinitionMap.get("parameterValues") != null) {
+                    ClassLoader parentLoader = Thread.currentThread().getContextClassLoader();
```

Comment:
Does this really work when run in workbench? @manstis could you comment on this cause I am not sure.